### DISCUSSION TITLE
请教一个同步词库的问题

### DISCUSSION THREAD

**Discussion by ShaneLee-9** [rime/home#1379] [Q&A]
如果我在两台设备上都有一个自定义词库hello.dict.yaml，但是里面的词条不同，点击同步后，rime如何合并这个文件的词条？根据我自己的测试结果，同步后我无法使用另一台设备的hello.dict.yaml词库中的词条

**Comment by Sunpulseqd** [comment]
RIME只同步用户自造词,不同步固定词库xxx.dict.yaml里的词.

**Comment by redleafnew** [comment]
我觉得同步用同步软件如坚果云、OD同步用户目录可以实现。
